api/_modules/user/user.repository.ts

```diff
@@ -0,0 +1,30 @@
+import { UserEntity } from './user.entity'
+import { logger } from 'api/_utils/logger'
+
+export class UserRepository {
+  async findByEmail(email: string) {
+    logger.debug('search for user in database', {
+      service: 'user.repository.findByEmail',
+      meta: { email }
+    })
+
+    const user = await UserEntity.findOneAsync({ email })
```

Comment:
Need to create a user type

```suggestion
    const user: UserType = await UserEntity.findOneAsync({ email })
```